magefile.go

```diff
@@ -148,3 +149,8 @@ func CheckLicenseHeaders() error {
 func DumpVariables() error {
 	return devtools.DumpVariables()
 }
+
+// UnitTest run unit tests (Go and Python) for all Beats
+func UnitTest() {
+	testtools.UnitTest()
+}
```

Comment:
This target will run all tests it finds with `go test`/`pytest` in the repository, this can be different to what the `make unit` test does, that loops over the list of projects.
If it works well I am on with the new approach, but just to be sure that we are aware of the possible differences.

Reply:
We could keep the `make unit` command as it is, reverting 54cd444d999570e4df070a57df715986d55d9e19, and start building a clear expectation from Mage side. I could think of a `test` namespace with:

- `test:unit`
- `test:integration`
- `test:system` 

Then we could add more child namespaces per beat 🤔

Reply:
> reverting 54cd444

Is this target needed for this change? I think 54cd444 is fine.

But if this target is intended to replace `make unit`, it should probably do the same, looping over the projects list and run `mage unitTest` for each one of them. The result of running `go test` and `pytest` over all the repository may have the same result, but not sure.

If it is not intended to replace `make unit`, and is not needed for other targets, I think we could remove it by now.

> I could think of a `test` namespace with:
> 
>   * `test:unit`
>   * `test:integration`
>   * `test:system`
> 
> Then we could add more child namespaces per beat thinking

Yep, there was a proposal about doing this kind of namespacing, but we never renamed all the targets. I would leave this for a future change by now.